src/Draco.Compiler.Tests/Syntax/ParserTests.cs

```diff
@@ -114,4 +220,1285 @@ public void TestEmptyFuncWithoutClosingCurly()
+                        this.T(TokenType.KeywordFalse);
+                    }
+                }
+                this.T(TokenType.KeywordOr);
+                this.N<Expr.Unary>();
+                {
+                    this.T(TokenType.KeywordNot);
+                    this.N<Expr.Literal>();
+                    {
+                        this.T(TokenType.KeywordFalse);
+                    }
+                }
+            }
+        }
+        this.VariableDeclarationPlaceHolder("""
+            true and false or not false
+            """, Operators);
+    }
+
+    [Fact]
+    public void TestOperatorGreaterThanPlusTimes()
+    {
+        void Operators()
+        {
+            this.N<Expr.Relational>();
+            {
+                this.N<Expr.Binary>();
+                {
+                    this.N<Expr.Literal>();
+                    {
+                        this.T(TokenType.LiteralInteger, "3");
+                    }
+                    this.T(TokenType.Plus);
+                    this.N<Expr.Literal>();
+                    {
+                        this.T(TokenType.LiteralInteger, "2");
+                    }
+                }
+                this.N<Expr.ComparisonElement>();
+                {
+                    this.T(TokenType.GreaterThan);
+                    this.N<Expr.Binary>();
+                    {
+                        this.N<Expr.Literal>();
+                        {
+                            this.T(TokenType.LiteralInteger, "2");
+                        }
+                        this.T(TokenType.Star);
+                        this.N<Expr.Literal>();
+                        {
+                            this.T(TokenType.LiteralInteger, "3");
+                        }
+                    }
+                }
+            }
+        }
+        this.VariableDeclarationPlaceHolder("""
+            3 + 2 > 2 * 3
+            """, Operators);
+    }
+
+    [Fact]
+    public void TestOperatorChainedRelations()
+    {
+        void Operators()
+        {
+            this.N<Expr.Binary>();
+            {
+                this.N<Expr.Relational>();
+                {
+                    this.N<Expr.Literal>();
+                    {
+                        this.T(TokenType.LiteralInteger, "3");
+                    }
+                    this.N<Expr.ComparisonElement>();
+                    {
+                        this.T(TokenType.GreaterThan);
+                        this.N<Expr.Literal>();
+                        {
+                            this.T(TokenType.LiteralInteger, "2");
+                        }
+                        this.N<ComparisonElement>();
+                        {
+                            this.T(TokenType.LessThan);
+                            this.N<Expr.Literal>();
+                            {
+                                this.T(TokenType.LiteralInteger, "8");
+                            }
+                        }
+                    }
```

Comment:
Comparison elements are not recursive, but sequential, so no need to nest it in the other. Other than that, looks correct!